get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,160 @@
+  </a>
+
+  <a  href="/layouts" class="btn-link-large">
+    Layouts
+    <i class="bi bi-layout-text-window-reverse primary" role="img" aria-hidden="true"></i>
+  </a>
+</div>
+
+## Reactivity {#reactivity}
+
+At the heart of Shiny is [reactivity](/docs/reactive-foundations.qmd), a system that automatically updates [outputs](/components/#outputs) when [inputs](/components/#inputs) change for seamless interactivity, without callbacks.
+
+Shiny's reactive engine avoids unnecessary computations by only re-calculating the outputs whose inputs have changed, making Shiny ideal for fast data-driven applications, enabling live updates for [charts, tables](/components/#outputs), and [reports](https://quarto.org/docs/interactive/shiny/) with minimal effort.
+
+Below is a Shiny application where you have the output text react to _both_ input slider values
+and the second row of text _only_ reacts to the first slider value.
+
+```{shinylive-python}
+#| standalone: true
+#| components: [editor, viewer]
+#| layout: vertical
+#| viewerHeight: 250
+
+from shiny import reactive
+from shiny.express import ui, render, input
+
+# input slider 1
+ui.input_slider("slider1", "Slider 1", min=1, max=50, value=20)
+
+# input slider 2
+ui.input_slider("slider2", "Slider 2", min=1, max=50, value=40)
+
+# output reacts to both sliders
+@render.text
+def both_sliders_output():
+    return f"You have set Slider 1: {input.slider1()}, Slider 2: {input.slider2()}"
+
+# output that only reacts to slider 1
+@render.text
+def result():
+    return f"The value for slider 1 squared: {input.slider1()**2}."
+```
+
+## Templates {#templates}
+
+Hit the ground running with `shiny create` to load a [starter template](/templates/) for common use cases, like [data dashboards](templates/dashboard/), [applications](/templates/database-explorer/), [streaming](/templates/monitor-database/) updates, or [data entry](/templates/survey/).
+
+Use the command below to [create](create-run.qmd) your first application.
+
+```bash
+shiny create --template dashboard
+```
+
+::::: {.column-screen-inset}
+::: {.hello-output}
+<iframe
+  src="https://gallery.shinyapps.io/template-dashboard/"
+  frameborder="0"
+  width="100%"
+  class="mb-0 card hello-output-iframe"
+></iframe>
+
+<style>
+  .hello-output-iframe {
+    height: 700px;
+  }
+@media only screen and (max-width: 611px) {
+  .hello-output-iframe {
+    height: 2125px;
+  }
+}
+</style>
+:::
+:::::
+
+
+## Extensibility {#extensible}
+
+While Shiny includes everything you need to build an app, is built on a foundation of web standards, making it highly [extensible](/docs/).
+
+If you need a [custom component](/docs/custom-components-pkg) or [user interface](/docs/ui-customize), you can incrementally integrate [JavaScript](/docs/custom-component-one-off), [HTML](/docs/ui-html.html), or even WebAssembly without having to learn complicated build tooling and frameworks.
```

Comment:
This should be at the end of the section after the "if you're versed in web programming" sentence